scripts/README.v1.1.sh

```diff
@@ -0,0 +1,90 @@
+###############################################################################
+# README file for how the calling regions for the single-sample HC based
+# exome calling pipeline were created.  The following is a two-part process
+# of identifying and formating various input sets of calling regions and then
+# secondarily combining and padding said regions.
+###############################################################################
+
+hg38=/seq/references/Homo_sapiens_assembly38/v0/
+picard=/seq/software/picard/1.1368/bin/picard-private-d5b94667a33ea4b110def476023b994de25868ef-all.jar
```

Comment:
This picard private is from Sep 6 2018.  It probably doesn't matter, but I did want to give a heads up.

Reply:
Thanks for pointing that out -- this should actually move to a separate PR because it isn't used here.